torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -110,7 +110,10 @@ void nnc_aten_adaptive_avg_pool2d(
   at::Tensor& r = tensors[0];
   const at::Tensor& x = tensors[1];
   int64_t H = extra_args[0];
-  int64_t W = extra_args[1];
+  int64_t W = H;
+  if (args_num > 1) {
```

Comment:
In my opinion, we should always pass in all of the args, and handle this kind of default propagation at the lowerings level.

Reply:
That sounds good to me, I will update the PR!